server/polar/member/service.py

```diff
@@ -167,5 +169,104 @@ async def list_by_customers(
         repository = MemberRepository.from_session(session)
         return await repository.list_by_customers(session, customer_ids)
 
+    async def create(
+        self,
+        session: AsyncSession,
+        auth_subject: AuthSubject[User | Organization],
+        *,
+        customer_id: UUID,
+        email: str,
+        name: str | None = None,
+        external_id: str | None = None,
+        role: MemberRole = MemberRole.member,
+    ) -> Member:
+        """
+        Create a new member for a customer.
+
+        Args:
+            session: Database session
+            auth_subject: Authenticated user/organization
+            customer_id: ID of the customer to add member to
+            email: Email address of the member
+            name: Optional name of the member
+            external_id: Optional external ID of the member
+            role: Role of the member (defaults to member)
+
+        Returns:
+            Created Member
+
+        Raises:
+            ResourceNotFound: If customer not found or not accessible
+            NotPermitted: If feature flag disabled or no permission to add members
+        """
+        repository = MemberRepository.from_session(session)
+        row = await repository.get_customer_with_organization(session, customer_id)
+
+        if row is None:
+            raise ResourceNotFound("Customer not found")
+
+        customer, organization = row
+
+        if not organization.feature_settings.get("member_model_enabled", False):
+            raise NotPermitted("Member management is not enabled for this organization")
```

Comment:
this is not following any of the patterns that we have in our code. We usually use customer.organizaiton